src/lib/blueprint.ts

```diff
@@ -189,19 +190,31 @@ const createEndpoint = (
   const pathParts = path.split('/')
   const endpointPath = `/${pathParts.slice(1, -1).join('/')}`
 
+  const description =
+    'description' in operation && typeof operation.description === 'string'
+      ? operation.description
+      : ''
+
+  const isUndocumented = 'x-undocumented' in operation
+  const deprecatedMatch = description.match(/---\s*deprecated:(.+?)---/s)
```

Comment:
Descriptions will not contain yaml front matter (they will always be parsed and stripped out).